gallery_dl/extractor/common.py

```diff
@@ -96,6 +96,17 @@ def request(self, url, *, method="GET", session=None, retries=None,
                         (400 <= code < 429 or 431 <= code < 500):
                     if encoding:
                         response.encoding = encoding
+
+                    if config.get((), "write_pages", False):
+                        # Write the response content to a .dump file
+                        # in the current directory.
+                        # The file name is derived from the response
+                        # url, replacing special characters with "_"
+                        r = re.compile(r'[:/.?&=#]+')
```

Comment:
```suggestion
                        r = re.compile(r"[\\\\|/<>:\"?*]")
```
Seems like last time I've managed to **not** click the "Add single comment" button. Lets try again ...

These are the same characters that are already used in [util.py](https://github.com/mikf/gallery-dl/blob/f557cac07405b48baabf316f595cbd538abb8807/gallery_dl/util.py#L674-L680) to produce valid filenames for Windows (ignoring any length limitations)

Reply:
Ok thanks, I've added the characters from util.py and kept excluding also &=#, which are annoying for the linux shell

Reply:
I mean, you'd **always** use tab auto-completion to "write" these filenames in your shell, whch takes care of escaping any special characters ...